src/App.tsx

```diff
@@ -1,173 +1,73 @@
-import { Loader } from './components/Loader';
-
+import React, { useEffect, useState } from 'react';
+import { Navigate, Route, Routes } from 'react-router-dom';
 import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import { Loader } from './components/Loader';
+import { LoadingError } from './LoadingError';
+import { MainNav } from './MainNav';
+import { PageNotFound } from './PageNotFound';
+import { PeoplePage } from './PeoplePage';
+import { Person } from './types';
+
+export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    const getUsers = async () => {
+      try {
+        setIsLoading(true);
+
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
+        );
+
+        const responseData = await response.json();
+
+        setPeople(responseData);
+      } catch (error) {
+        // eslint-disable-next-line no-alert
+        alert('error');
```

Comment:
Pls use useState to store an error and show it somewhere, don't use alerts/console.logs for such cases

Reply:
424267880fd4e2eecbbd66c8c9a37fbb6d7063cc